h/templates/client/about_this_version_dialog.html

```diff
@@ -0,0 +1,25 @@
+<div class="sheet">
+  <i class="close h-icon-close"
+    role="button"
+    title="Close"
+    ng-click="vm.aboutThisVersionDialog.visible = false"></i>
```

Comment:
As a general principle, new components should not directly manipulate data that is passed in via the scope but rather emit events when the user performs some action, and the parent handles the change.

So in this case, rather than the dialog itself setting the flag which indicates which dialog is visible, it would have an `onClose` event which is triggered when the user clicks the 'Close' button. https://github.com/hypothesis/h/pull/3215/files has a rewrite of the existing `<share-dialog>` to follow this pattern.